src/Yoti.Auth/Web/SignedMessageFactory.cs

```diff
@@ -13,7 +13,7 @@ public static string SignMessage(HttpMethod httpMethod, string endpoint, Asymmet
                 throw new ArgumentNullException(nameof(httpMethod));
             }
 
-            string stringToConvert = $"{httpMethod.ToString()}&{endpoint}";
+            string stringToConvert = $"{httpMethod}&{endpoint}";
```

Comment:
Presume this was an unnecessary cast?

Reply:
yes indeed totally unecessary ToString() call inside interpolation, plus was kicking out warning.